pages/loginPage.ts

```diff
@@ -0,0 +1,81 @@
+import { expect, Locator, Page } from '@playwright/test';
+
+require('dotenv').config();
+
+
+export class LoginPage {
+
+    readonly page: Page;
+    readonly createAccount: Locator;
+    readonly firstName: Locator;
+    readonly lastName: Locator;
+    readonly email: Locator;
+    readonly password: Locator;
+    readonly confirmPassword: Locator;
+    readonly createAccountBtn: Locator;
+    readonly consent: Locator;
+
+
+
+    constructor(page: Page) {
+        this.page = page;
+        this.createAccount = page.getByRole('link', { name: 'Create an Account' });
+        this.firstName = page.getByLabel('First Name');
+        this.lastName = page.getByLabel('Last Name');
+        this.email = page.getByLabel('Email', { exact: true });
+        this.password = page.getByRole('textbox', { name: 'Password*', exact: true });
+        this.confirmPassword = page.getByLabel('Confirm Password');
+        this.createAccountBtn = page.getByRole('button', { name: 'Create an Account' });
+        this.consent = page.getByLabel('Consent', { exact: true });
+
+    }
+
+    async navigate() {
+        await this.page.goto(process.env.URL as string);
```

Comment:
```suggestion
        await this.page.goto('/');
```
I believe that it should navigate the user to the baseURL provided in playwright config.

Reply:
Thank you Eleonora for the feedback , it is very useful , I just addressed the feedback